woocommerce/woocommerce
# update/a2p update `api-core-tests` to run against both CI test env and JN

### All Submissions:






### Changes proposed in this Pull Request:


update `api-core-tests` to run against both CI test env and JN
update settings test to ensure it executes correctly

Closes 423-gh-woocommerce/woocommerce-quality and 426-gh-woocommerce/woocommerce-quality






### How to test the changes in this Pull Request:



1. Pull down the code
2. Navigate to woocommerce/plugins/woocommerce
3. Execute tests with pnpm test:api-pw
4. All tests should execute and pass successfully
5. Create JN site
6. Install the Basic Auth plugin (from  https://github.com/WP-API/Basic-Auth/archive/master.zip) so you could send REST API requests to this JN site
7. Activate the Basic Auth plugin. 
8. Obtain BASE_URL, USER_KEY and USER_SECRET values from JN
9. Create .env file in the `woocommerce/plugins/woocommerce` directory in the following format and update variables with those obtained from JN in step 7:
BASE_URL = "https://##########.jurassic.ninja"
USER_KEY = "#######"
USER_SECRET = "###########"
10. Execute tests with pnpm test:api-pw
11. All tests should execute against JN environment and pass successfully



### Other information:




### FOR PR REVIEWER ONLY:
